tests/10-migration-backend-tests/testcafe/migrations/media-accessrights.csv

```diff
@@ -0,0 +1,2 @@
+unique_id,name,authority,description
```

Comment:
This is a new migration that is being added by this PR. Not yet sure how it fits into the bigger picture.  Were access rights not being tested before this PR?